obelisk_ws/src/cpp/obelisk_leap_cpp/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.16)
+project(obelisk_leap_cpp)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(dynamixel_sdk REQUIRED)
+
+# Get the Obelisk library
+include(FetchContent)
+FetchContent_Declare(
+  Obelisk
+  SOURCE_DIR $ENV{OBELISK_ROOT}/obelisk/cpp
+)
+
+# Lets us get Obelisk::Core and Obelisk::Zoo
+FetchContent_MakeAvailable(Obelisk)
+
+set(LD_LIBRARY_PATH "/home/gavin/caltech/surf2024/obelisk/.pixi/envs/dev/lib:$ENV{LD_LIBRARY_PATH}")
```

Comment:
this LD_LIBRARY_PATH is hardcoded to your local filesystem path - you can remove this line, since we patched the fix on the pixi side for needing to edit the LD_LIBRARY_PATH already.